parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -661,7 +661,7 @@ configGPUDrivers() {
             fi
             docker rmi $NVIDIA_DRIVER_IMAGE:$NVIDIA_DRIVER_IMAGE_TAG
         fi
-    elif [[ $OS == $MARINER_OS_NAME ]]; then
+    elif isMarinerOrAzureLinux "$OS"; then
         downloadGPUDrivers
         installNvidiaContainerRuntime
```

Comment:
I am a bit confused. In the PR description it says `Do not need to install nvidia-container-runtime package for Azure Linux 3.0`. But this is still installing?

Reply:
Oh, sorry for the confusion. What I meant was that, for Azure Linux 3.0, we do not need to install the `nvidia-container-runtime` package in particular but still need the other nvidia packages that are installed in the `installNvidiaContainerRuntime` function.

The `installNvidiaContainerRuntime` function installs multiple nvidia packages: `nvidia-container-toolkit`, `nvidia-container-toolkit-base`, `libnvidia-container-tools`, `libnvidia-container1`

Talked with @henryli001 and I will update the function name to `installNvidiaConainerToolkit` to make this clearer so there is no confusion.

Reply:
Updated the function name - [commit](https://github.com/Azure/AgentBaker/pull/4808/commits/19a4ebd23763fd91857c63d0be9beb8fda882ee7)

Hopefully, this helps here :)